### ISSUE TITLE
Allow reading from `stdin`

### ISSUE THREAD

**Issue by sdroege** [Lyude/neovim-gtk#57]
`gvim` allows to do that by passing `-` as filename

**Comment by Lyude** [comment]
Yeah - the way we handle this currently is leftover from daa84's version, and it's definitely quite silly seeing as nvim has support for proper pipes.
